Dockerfiles/midstream/Dockerfile

```diff
@@ -11,6 +12,8 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     curl -fL -o /usr/local/bin/umoci https://github.com/opencontainers/umoci/releases/download/${UMOCI_VERSION}/umoci.amd64 && \
     chmod a+x /usr/local/bin/umoci && \
     mkdir /project/output && \
+    chmod g+x /entrypoint.sh && \
```

Comment:
You do not need such step if file bits are already properly set in git repository

Reply:
updated